db/schema.rb

```diff
@@ -189,23 +188,6 @@
     t.datetime "updated_at",     :null => false
   end
 
-  create_table "notifications", :force => true do |t|
```

Comment:
I'm confused by these changes to schema. It looks like you are removing multiple tables that deal with notifying people ?